src/pages/MonthlyGood/ExpandableQuestion.vue

```diff
@@ -0,0 +1,89 @@
+<template>
```

Comment:
What do you think about making this into a re-usable KvAccordian component?

Reply:
Lets hold off on this for now

Reply:
Agree. we should build at some point. @ryan-ludwig could you please create a ticket in the vue queue for this new component... you could ref this file as an example of what should be supported.